source/NVDAObjects/IAccessible/adobeAcrobat.py

```diff
@@ -137,15 +143,30 @@ def _getNodeMathMl(self, node):
 		yield "</%s>" % tag
 
 	def _get_mathMl(self):
+		if self.pdDomNode is None:
+			log.debugWarning("_get_mathMl: self.pdDomNode is None!")
+			raise LookupError
 		# There could be other stuff before the math element. Ug.
+		mathMl = self.pdDomNode.GetValue()
+		log.debug(
+			f'\n_get_mathMl: math recognized: {mathMl.startswith("<math")}, child count={self.pdDomNode.GetChildCount()}',
+		)
+		log.debug(f"\nname={self.pdDomNode.GetName()}\nvalue={self.pdDomNode.GetValue()}")
```

Comment:
I don't think a new line should be added here, won't this just create an unusual new line split between the log time and thread prefix and the log message.
If you want to group the log messages and have new lines, please join the messages together

```suggestion
		log.debug(
			f'\n_get_mathMl: math recognized: {mathMl.startswith("<math")}, child count={self.pdDomNode.GetChildCount()}',
			f"\nname={self.pdDomNode.GetName()}\nvalue={self.pdDomNode.GetValue()}"
		)
```